discover/index.html

```diff
@@ -58,41 +58,41 @@ <h3 class="text-left" id="connect"><i class="fab fa-fw fa-mixcloud"></i> Get Con
     <ul class="dotless">
-      <li><em>Available Years: 2011 - 2016</em>
+      <li><em>Available Years: 2011–2016</em>
     </ul>
   <li><i class="fas fa-fw fa-unlock"></i> <code>Basemap-Address_Points</code> – Base map of address point locations
-  <li><i class="fas fa-fw fa-lock"></i> <code>Basemap-Hybrid</code> - Base map combination of the Overlay base map and Utah imagery service <strong>(licensed)</strong>
-  <li><i class="fas fa-fw fa-unlock"></i> <code>Basemap-Lite</code> – Base map of hillshade, HWYs/streets, cities, boundaries, etc. (gray, muted tone)
-  <li><i class="fas fa-fw fa-unlock"></i> <code>Basemap-Overlay</code> – Base map of vector features HWYs/streets, cities, address points, parcels, etc. (transparent background)
-  <li><i class="fas fa-fw fa-unlock"></i> <code>Basemap-Terrain</code> – Base map of hillshade, HWYS/streets, parks, forests, water, etc.
-  <li><i class="fas fa-fw fa-unlock"></i> <code>Basemap-Topo</code> – Base map of USGS topographic maps
-  <li><i class="fas fa-fw fa-unlock"></i> <code>DOQ 1990s BW</code> – 1-meter statewide black &amp; white aerial photography collected in 1990’s
-  <li><i class="fas fa-fw fa-unlock"></i> <code>HRO 2012 RGB</code> – 12.5-cm natural color aerial photography collected in 2012 of the Wasatch Front
-  <li><i class="fas fa-fw fa-unlock"></i> <code>NAIP {year} RGB</code> – 1-meter statewide natural color aerial photography
+  <li><i class="fas fa-fw fa-lock"></i> <code>Basemap-Hybrid</code> - Base map combination of the overlay base map and Utah imagery service <strong>(licensed)</strong>
+  <li><i class="fas fa-fw fa-unlock"></i> <code>Basemap-Lite</code> – Base map of hillshade, highways and streets, cities, boundaries, etc. (gray, muted tone)
+  <li><i class="fas fa-fw fa-unlock"></i> <code>Basemap-Overlay</code> – Base map of vector features, highways and streets, cities, address points, parcels, etc. (transparent background)
+  <li><i class="fas fa-fw fa-unlock"></i> <code>Basemap-Terrain</code> – Base map of hillshade, highways and streets, parks, forests, water, etc.
+  <li><i class="fas fa-fw fa-unlock"></i> <code>Basemap-Topo</code> – Base map of US Geological Survey topographic maps
+  <li><i class="fas fa-fw fa-unlock"></i> <code>DOQ 1990s BW</code> – 1-meter statewide black-and-white aerial photography, collected in the 1990s
+  <li><i class="fas fa-fw fa-unlock"></i> <code>HRO 2012 RGB</code> – 12.5-cm natural-color aerial photography of the Wasatch Front, collected in 2012 
+  <li><i class="fas fa-fw fa-unlock"></i> <code>NAIP {year} RGB</code> – 1-meter statewide natural-color aerial photography
     <ul class="dotless">
-      <li><em>Available Years: 2006 2009 2011 2014 2016</em>
+      <li><em>Available Years: 2006, 2009, 2011, 2014, 2016</em>
     </ul>
   <li><i class="fas fa-fw fa-unlock"></i> <code>NAIP {year} NRG</code> – 1-meter statewide color infrared aerial photography
```

Comment:
"color infrared"--is this correct? Just making sure. It sounds a bit off to me...

Reply:
yes color infrared changes the colors https://en.wikipedia.org/wiki/Infrared_photography

Reply:
Interesting! Learn something new every day! :) K, we'll leave it then.